src/DnsSecStack.ts

```diff
@@ -1,11 +1,19 @@
 import * as cdk from 'aws-cdk-lib';
-import { aws_ssm as SSM, Tags, aws_iam as IAM, aws_kms as KMS } from 'aws-cdk-lib';
+import { aws_ssm as SSM, Tags, aws_iam as IAM, aws_kms as KMS, aws_route53 as route53 } from 'aws-cdk-lib';
 import { Construct } from 'constructs';
 import { Statics } from './Statics';
 
+export interface DnsSecStackProps extends cdk.StackProps {
+  /**
+   * If this stack is created it creats a KSM key
```

Comment:
KMS?

Reply:
KMS key die wordt gebruikt voor het maken van een DNSSEC Key signing key

Reply:
Sorry, ik bedoelde de tikfout in de comment. ;)